.github/actions/test-pylint/action.yml

```diff
@@ -13,6 +13,10 @@ runs:
         sudo apt-get -qq update
         sudo apt-get -qq -y install python3-pip gnupg
         python -m pip install --upgrade pip
+    - name: Set up Python 3.11
+      uses: actions/setup-python@v5
+      with:
+        python-version: "3.11"
```

Comment:
What is this used for?
Can you add a comment why we need this, and why it's only 3.11 (i.e. our "latest")? So we know when to upgrade this.

Maybe we could add a variable for our python versions somewhere, but I'd be fine with postponing that.

Reply:
i have added a comment. THe idea with the variable for the version is amazing :) However, having no idea right now, how to realize this (and where)? WIll keep this in mind and maybe do it in the future.